packages/frontend-2/components/pricingTable/PricingTable.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="flex flex-col lg:grid lg:grid-cols-3 gap-4 w-full">
+    <PricingTablePlan
+      v-for="plan in plans"
+      :key="plan"
+      :plan="plan"
+      :current-plan="currentPlan"
+      :yearly-interval-selected="isYearlySelected"
+      :active-billing-interval="billingInterval"
+      :is-admin="isAdmin"
+      :workspace-id="props.workspaceId"
+      @on-yearly-interval-selected="onYearlyIntervalSelected"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { BillingInterval } from '~/lib/common/generated/gql/graphql'
+import { WorkspacePlans } from '@speckle/shared'
+import { useWorkspacePlan } from '~~/lib/workspaces/composables/plan'
+import { graphql } from '~/lib/common/generated/gql'
+import { type MaybeNullOrUndefined, type WorkspaceRoles, Roles } from '@speckle/shared'
+
+graphql(`
+  fragment PricingTable_Workspace on Workspace {
+    id
+    role
+  }
+`)
+
+const props = defineProps<{
+  slug: string
+  role: MaybeNullOrUndefined<WorkspaceRoles>
```

Comment:
I will probably move all of these into a composable, as mentioned earlier in another PR, but I will do this as a follow-up so I can adjust it in all places